pkg/virt-handler/vm_test.go

```diff
@@ -3266,7 +3275,12 @@ var _ = Describe("VirtualMachineInstance", func() {
 		})
 
 		It("should succeed with empty VMI and all commands", func() {
-			result, reason := isGuestAgentSupported(vmi, allCommands)
+			var commands []v1.GuestAgentCommandInfo
+			commands = append(commands, basicCommands...)
```

Comment:
You could define/initialize the `commands` slices directly, no?

Reply:
I think `commands := append(basicCommands, sshCommands...)` would modify the underlying storage of `basicCommands`. 
In general, I don't like using `append` with 2 different slice variables: `a := append(b, c)`, because then slices `a` and `b` share the same elements in memory, and modifications using `a[i]` are reflected in `b[i]` which can be confusing.

But it this cas for the test it is safe. If you want, I can rewrite it as:
```go
commands := append(basicCommands, sshCommands...)
commands = append(commands, passwordCommands...)
```

Reply:
I see! I was thinking of `commands := []v1.GuestAgentCommandInfo{basicCommands..., sshCommands..., passwordCommands...`, but that doesn't work, right? Just ignore this then.

Reply:
Right, it doesn't compile.